src/login/Login.jsx

```diff
@@ -9,6 +9,7 @@ export default function Login() {
     const [errorMessage, setErrorMessage] = useState(null);
     const {isAuth,login} = useContext(AuthContext);
     const [user, setUser] = useState(null);
+    const [isPasswordShown, setIsPasswordShown] = useState(false)
     
     const handleSubmit = async e => {
         e.preventDefault();
```

Comment:
@ruppysuppy . I've tested the change you've made. The toggle checkbox has unintentionally got included inside the payload body and causing an error response `Wrong username and password` even though it was using the correct combo.
Here's the screenshot UI response:
<img width="1306" alt="Screen Shot 2020-09-10 at 10 13 14 am" src="https://user-images.githubusercontent.com/29667122/92667558-93736280-f34f-11ea-844c-8568646c0cec.png">

Here're my findings from debugging:
* incorrect payload body caused by code change
<img width="1680" alt="Screen Shot 2020-09-10 at 10 15 08 am" src="https://user-images.githubusercontent.com/29667122/92667651-daf9ee80-f34f-11ea-96c8-e8fa4d04e3dc.png">

* the expected payload as per tested on deevelop branch
<img width="1680" alt="Screen Shot 2020-09-10 at 10 17 16 am" src="https://user-images.githubusercontent.com/29667122/92667705-fe249e00-f34f-11ea-923d-1fe0ed1042c4.png">

You can fix this by filtering out the checkbox from being included in the payload.